.circleci/config.yml

```diff
@@ -73,6 +79,7 @@ jobs:
                   command: |
                       . venv/bin/activate
                       mkdir -p results
+                      export NO_ML=true
                       pytest --cov=wandb --junitxml="results/pytest.xml" tests/test_headless.py tests/test_cli.py tests/test_wandb.py
```

Comment:
These are general improvements to circle.  Added python 3.7, standardized on the dockerhub images, and simplified ml installation.